src/Facades/Saloon.php

```diff
@@ -13,6 +13,11 @@
  * @method assertSentJson(string $request, array $data)
  * @method assertNothingSent()
  * @method assertSentCount(int $count)
+ * @method record()
+ * @method stopRecording()
+ * @method isRecording()
+ * @method getRecordedResponses()
+ * @method getLastRecordedResponse()
```

Comment:
all of these Facade method docs should be `@method static` as they are called `Saloon::record()`.

Reply:
And it would be great if there could be return type hints by using `@method static ReturnType method()`